ale/formatters/usgscsm_formatter.py

```diff
@@ -51,6 +51,9 @@ def to_usgscsm(driver):
         'unit' : 'm'
     }
 
+    isd_data["projection"] = driver.projection
+    isd_data["geotransform"] = driver.geotransform
```

Comment:
This needs to make it's way into the `to_isd` function in the generic formatter.